src/index.rs

```diff
@@ -1,11 +1,12 @@
 use std::{
-    collections::{HashMap, HashSet},
+    borrow::Cow,
     fmt::{Debug, Formatter},
     hash::Hash,
     usize,
 };
 
-use crate::{FieldAccessor, Filter, Tokenizer};
+use crate::{FieldAccessor, Tokenizer};
+use hashbrown::{HashMap, HashSet};
```

Comment:
That;s quite a significant change in performance, I guess most of it is the switch to `hashbrown`?

I can imagine a small gain from removing the `Filter` calls but not that significant.

Reply:
Yes. Most is hashbrown . A small portion seem to be from the Cow change in the ```add_document``` also, but much less significant